go/vt/mysqlctl/backupstorage/interface.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package backupstorage contains the interface and file system implementation
+// of the backup system.
+package backupstorage
+
+import (
+	"flag"
+	"io"
+
+	log "github.com/golang/glog"
+)
+
+var (
+	// BackupStorageImplementation is the implementation to use
+	// for BackupStorage. Exported for test purposes.
+	BackupStorageImplementation = flag.String("backup_storage_implementation", "", "which implementation to use for the backup storage feature")
+)
+
+// BackupHandle describes an individual backup.
+type BackupHandle interface {
+	// Bucket is the location of the backup. Will contain keyspace/shard.
+	Bucket() string
+
+	// Name is the individual name of the backup. Will contain
+	// tabletAlias-timestamp.
```

Comment:
Is there another character that's safe to use besides '-'? That's already used to separate elements within a tabletAlias, and within a date. It makes for a name that's somewhat human-unfriendly: cell-0001-2015-01-14-10-00-00

As a human, I'd prefer to see something like: cell-0001.2015-01-14.100000